_search-plugins/star-tree-index.md

```diff
@@ -7,77 +7,133 @@ nav_order: 54
 
 # Star-tree index
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
-{: .warning}
+A _star-tree index_ is a multi-field index that improves the performance of aggregations by precomputing metric values for combinations of dimension fields.
 
-A star-tree index is a multi-field index that improves the performance of aggregations.
+Once you enable star-tree indexes, OpenSearch automatically builds and uses star-tree indexes to optimize aggregations if the filter fields match the defined dimensions and the aggregation fields match the defined metrics in the star-tree configuration. No changes to your query syntax or request parameters are required.
 
-OpenSearch will automatically use a star-tree index to optimize aggregations if the queried fields are part of dimension fields and the aggregations are on star-tree metric fields. No changes are required in the query syntax or the request parameters.
-
-## When to use a star-tree index
-
-A star-tree index can be used to perform faster aggregations. Consider the following criteria and features when deciding to use a star-tree index:
+Use a star-tree index when you want to speed up aggregations:
 
 - Star-tree indexes natively support multi-field aggregations.
-- Star-tree indexes are created in real time as part of the indexing process, so the data in a star-tree will always be up to date.
-- A star-tree index consolidates data, increasing index paging efficiency and using less IO for search queries.
+- Star-tree indexes are created in real time as part of the indexing process, so the data in a star-tree is always current.
+- A star-tree index consolidates data to improve paging efficiency and reduce disk I/O during search queries.
 
-## Limitations
+## Star-tree index structure
 
-Star-tree indexes have the following limitations:
+A star-tree index organizes and aggregates data across combinations of dimension fields and precomputes metric values for those combinations during ingestion. This structure enables OpenSearch to process aggregation queries quickly without scanning every document.
```

Comment:
> during ingestion

every time a segment is flushed can be more descriptive.

Reply:
* refreshed / flushed